### PR TITLE
Make investor pane placement make more sense

### PR DESCRIPTION
**Pull request by Xynth22**
Change from Bunnywabbit to make investor pane placement make more sense

### CODE REVIEW

**Comment by Xynth22** [comment]
@hheimbuerger Cort, can you double check this. Link to forum discussion, I am merely the messenger.

https://www.freeallegiance.org/forums/index.php?showtopic=72367

@BackTrak

File: dialog.mdl [hunk 1]
```diff
@@ -5372,7 +5372,9 @@ dialogs =
         (TeamPaneImage, SideRight, Point(10, -150), Point(-750, -150), .5, ofTeam | ofExpandedTeam, false),
 
         // MergedInvestmentsPaneImage: the investments pane (F5)
-        (MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
+        //Change from Bunnywabbit to make investor pane placement make more sense
```
Comment (hheimbuerger) [inline dialog.mdl]: @Xynth22 I'm in favor of no longer having historical information stored in the code. That's what git is great for.

Let's remove lines 5375 and 5376.

File: dialog.mdl [hunk 2]
```diff
@@ -5372,7 +5372,9 @@ dialogs =
         (TeamPaneImage, SideRight, Point(10, -150), Point(-750, -150), .5, ofTeam | ofExpandedTeam, false),
 
         // MergedInvestmentsPaneImage: the investments pane (F5)
-        (MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
+        //Change from Bunnywabbit to make investor pane placement make more sense
+        //(MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
+        (MergedInvestmentsPaneImage, SideRight, Point(10, -265), Point(-490, -265), .2, ofInvestment, false),
```
Comment (hheimbuerger) [inline dialog.mdl]: I think we should keep the animations consistent. I find it subconsciously confusing if every window has a different animation (length, positioning, etc.). Therefore, I suggest we keep this animation at `.5`s.

**Review by hheimbuerger — CHANGES_REQUESTED** [review]
Besides my change requests on individual lines, I think this is a great idea and should be merged!

**Comment by Xynth22** [comment]
incorporated Cort's comments. Ready to go!

File: dialog.mdl [hunk 3]
```diff
@@ -5372,7 +5372,8 @@ dialogs =
         (TeamPaneImage, SideRight, Point(10, -150), Point(-750, -150), .5, ofTeam | ofExpandedTeam, false),
 
         // MergedInvestmentsPaneImage: the investments pane (F5)
-        (MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
+        //(MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
```
Comment (hheimbuerger) [inline dialog.mdl]: No need to keep this comment! History's in git if someone wants to go back.

**Review by hheimbuerger — CHANGES_REQUESTED** [review]

**Comment by Xynth22** [comment]
Ah that does look better in Git when you do it that way.

File: dialog.mdl [hunk 4]
```diff
@@ -5371,8 +5371,8 @@ dialogs =
         // TeamPaneImage: the information window about the teams (F6)
         (TeamPaneImage, SideRight, Point(10, -150), Point(-750, -150), .5, ofTeam | ofExpandedTeam, false),
 
-        // MergedInvestmentsPaneImage: the investments pane (F5)
-        (MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
+        // MergedInvestmentsPaneImage: the investments pane (F5)
```
Comment (hheimbuerger) [inline dialog.mdl]: You added a whole bunch of spaces at the end of this line. :)

**Review by hheimbuerger — APPROVED** [review]

**Comment by Xynth22** [comment]
Done.

**Comment by BackTrak** [comment]
sorry guys, totally missed that there were pulls waiting in artwork!

**Pull request merged**